ThunderCloud/Quiz.swift

```diff
@@ -108,8 +115,14 @@ open class QuizPage: StormObjectProtocol {
             if QuizConfiguration.shared.shuffleQuestions {
                 questions.shuffle()
             }
-            
-            self.questions = questions
+
+            if QuizConfiguration.shared.hasMaxNumberOfQuestions {
+                self.questions = Array(
+                    questions.prefix(QuizConfiguration.shared.maxNumberOfQuestions ?? 5)
```

Comment:
5 is far too opinionated for a framework, when we make my `maxNumberOfQuestions` anyway this will fall in to place